coalib/parsing/Globbing.py

```diff
@@ -191,7 +191,7 @@ def translate(pattern):
                 regex += '[' + sequence + ']'
         else:
             regex = regex + re.escape(char)
-    return regex + '\\Z(?ms)'
+    return '(?ms)' + regex + '\\Z'
```

Comment:
AFAIK it doesn't matter where those inline `(?flags)` are located in the regex string